server/api/cartItem.js

```diff
@@ -0,0 +1,64 @@
+const router = require("express").Router()
+const {models : {CartItem, Product}} = require("../db")
+
+router.get("/allCartItems/:userId", async (req, res, next) => {
```

Comment:
1.  change to "/carts/:userId"
2.  add comments above routes describing the route => // GET /api/carts/:userId